polygeist/tools/cgeist/Lib/CodeGenTypes.cc

```diff
@@ -195,8 +195,11 @@ mlir::Type CodeGenTypes::getMLIRType(clang::QualType qt, bool *implicitRef,
           TypeName == "group") {
         return getSYCLType(RT, *this);
       }
-      llvm::errs() << "Warning: SYCL type '" << ST->getName()
-                   << "' has not been converted to SYCL MLIR\n";
+      // No need special handling for types that doesn't have record declaration
+      // name.
```

Comment:
Can you give an example of such a type, which doesn't have a `TypeName`?

Reply:
There is an anonymous union type declared inside the `accessor` class, so:
```
Warning: SYCL type 'union.anon' has not been converted to SYCL MLIR
```
Is shown.